addons/docs/src/frameworks/svelte/svelte-docgen-loader.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-param-reassign */
+// @ts-ignore
```

Comment:
add to typings.d.ts instead of ts-ignore

Reply:
i added a typings.d.ts file and removed the // @ts-ignore comment